aasemble/django/apps/mirrorsvc/models.py

```diff
@@ -125,6 +125,7 @@ class Snapshot(models.Model):
     uuid = models.UUIDField(unique=True, default=uuid.uuid4, editable=False)
     timestamp = models.DateTimeField(auto_now_add=True)
     mirrorset = models.ForeignKey(MirrorSet)
+    tag = models.CharField(max_length=200)
```

Comment:
No, you can expose them however you please. The django-rest-framework docs have several examples.